src/Lumi/Components/Input.purs

```diff
@@ -173,6 +174,32 @@ text_ =
   , variant: None
   }
 
+inputPlaceholderText_ :: String -> InputProps
+inputPlaceholderText_ s =
```

Comment:
I'd say to this the same that Harry's commented about `textareaPlaceholderText_`: don't think we need a separate function for this, as this is equivalent to `text_ { placeholder = s }`.

Reply:
I think @arthurxavierx was suggesting removing this function entirely rather than just simplifying it by expressing it in terms of `text_`.

Reply:
Removed.